packages/core/admin/admin/src/features/StrapiApp.tsx

```diff
@@ -8,16 +9,15 @@ import type { StrapiApp } from '../StrapiApp';
  * -----------------------------------------------------------------------------------------------*/
 interface StrapiAppContextValue
   extends Pick<
-    StrapiApp,
-    | 'customFields'
-    | 'menu'
-    | 'getPlugin'
-    | 'getAdminInjectedComponents'
-    | 'plugins'
-    | 'runHookParallel'
-    | 'runHookSeries'
-    | 'settings'
-  > {
+      StrapiApp,
+      | 'customFields'
+      | 'getPlugin'
+      | 'getAdminInjectedComponents'
+      | 'plugins'
+      | 'runHookParallel'
+      | 'runHookSeries'
+    >,
+    Pick<Router, 'menu' | 'settings'> {
```

Comment:
Why do we want to leave menu and settings at the root level here if they're actually in router and addressed that way everywhere?

Reply:
to avoid more BC. I'd rather deprecate, which i can do now actually – than add more BCs just for the fun of it lol wdyt?